test/integration/TokenTransfers.spec.ts

```diff
@@ -0,0 +1,120 @@
+import 'mocha'
+import { expect } from 'chai'
+
+import {
+  radixUniverse,
+  RadixUniverse,
+  RadixIdentityManager,
+  RadixTransactionBuilder,
+  RadixAccount,
+  RadixLogger,
+  logger,
+  RadixIdentity,
+} from '../../src'
+
+const ERROR_MESSAGE = 'Local node needs to be running to run these tests'
+
+describe('Token transfers', () => {
+    const identityManager = new RadixIdentityManager()
+
+    let identity1: RadixIdentity
+    let account2: RadixAccount
+    let TBD_URI: string
```

Comment:
Nitpicking, but technically this isn't a `URI`, but rather an `RRI`.
(URI = `scheme:[//authority]path[?query][#fragment]`)

Reply:
Hmm, it seems to me that a URI is a more broad category that applies to any Unique Resource Identifiers, what you have specified is a URL scheme https://tools.ietf.org/html/rfc3986#section-1.1.2

Reply:
Yes, that is correct; URL and URN are subsets of URI, but that was not my point.

My point was that a `URI` requires a `scheme` and `TBD_URI` never defines a `scheme` - we don't have a `radix:` scheme, or any other scheme for that matter, for any of our resources, hence, technically speaking, they are not `URI`'s.

But again, this is all nitpicking and nothing that should be taken too seriously.